proto/replicationdata.proto

```diff
@@ -38,3 +38,16 @@ message Status {
   uint32 master_server_id = 11;
   string master_uuid = 12;
 }
+
+// StopReplicationStatus represents the slave status before calling stop slave, and the slave status collected immediately after
```

Comment:
reword to avoid "slave". We should use replication / replica as appropriate.

Reply:
I've reworded as much of it as I can without obfuscating literal MySQL statements that are intended to provide clarity for developers.